lib/super_good/solidus_taxjar/discount_calculator.rb

```diff
@@ -6,7 +6,7 @@ def initialize(line_item)
       end
 
       def discount
-        -line_item.promo_total
+        -(line_item.promo_total + line_item.adjustments.select { |value| !value.tax? && value.eligible? }.sum(&:amount))
```

Comment:
Me and @nvandoorn expanded on the tests added here and discovered that this will double count promotion adjustments since they are already included in the `promo_total`. We have an updated version of this change which completely removes the `promo_total` and just does the sum of the adjustments in Ruby which may be closer to what we want.